README.md

```diff
@@ -449,27 +449,31 @@ This updates core `logdize_logger` DB function. No need to update tables or trig
 If you want to update Logidze settings for the model, run migration with `--update` flag:
 
 ```sh
-rails generate logidze:model Post --update --only=title,body,rating
+bundle exec rails generate logidze:model Post --update --only=title,body,rating
 ```
 
 You can also use the `--name` option to specify the migration name to avoid duplicate migration names:
 
 ```sh
-$ rails generate logidze:model Post --update --only=title,body,rating --name add_only_filter_to_posts_log_data
+$ bundle exec rails generate logidze:model Post --update --only=title,body,rating --name add_only_filter_to_posts_log_data
 
     create db/migrate/20202309142344_add_only_filter_to_posts_log_data.rb
 ```
 
+### Pending upgrade check
```

Comment:
```suggestion
### Pending upgrade check [Experimental]
```

And let's also clearly state that it is disabled by default.

Reply:
Sure! Also mentioned it in the changelog.